mypy/options.py

```diff
@@ -113,6 +117,9 @@ def __init__(self) -> None:
         self.debug_cache = False
         self.quick_and_dirty = False
 
+        # plugins
+        self.plugins = []  # type: List[PluginRegistry]
```

Comment:
Thanks for the heads up.  I suspected a cycle too, but knew that they seemed to be working in general.  I'll see if I can resolve the cycle elsewhere.